frontend/src/lib/components/LemonTable/columnUtils.tsx

```diff
@@ -7,11 +7,11 @@ import { UserBasicType } from '~/types'
 
 export function createdAtColumn<T extends { created_at?: string | null }>(): LemonTableColumn<T, 'created_at'> {
     return {
-        title: 'Created on',
+        title: 'Created',
         dataIndex: 'created_at',
         render: function RenderCreatedAt(created_at) {
             return created_at ? (
-                <div style={{ whiteSpace: 'nowrap' }}>
+                <div style={{ whiteSpace: 'nowrap', textAlign: 'center' }}>
```

Comment:
Why was this changed to centered?

Reply:
because it looks really horrible when it's right aligned completely as there's extra padding/considerations that'll be needed to offset it

<img width="217" alt="Screen Shot 2022-09-02 at 12 03 03 PM" src="https://user-images.githubusercontent.com/25164963/188193948-ddc7eb53-80f3-463e-bc39-a78e209c6221.png">

Reply:
flyby: I think if both the header and column are right-aligned, it'll look just fine...

Reply:
Yeah for it to work the column header and contents should all have the same alignment. 
<img width="601" alt="Screen Shot 2022-09-02 at 9 16 57 AM" src="https://user-images.githubusercontent.com/254612/188196156-c68f55e0-7248-4828-893b-19ec6bbfcb8f.png">